packages/block-editor/src/components/block-styles/index.js

```diff
@@ -105,37 +108,57 @@ function BlockStyles( {
 				return (
 					<div
 						key={ style.name }
-						className={ classnames(
-							'editor-block-styles__item block-editor-block-styles__item', {
-								'is-active': activeStyle === style,
-							}
-						) }
-						onClick={ () => updateClassName( style ) }
-						onKeyDown={ ( event ) => {
-							if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
-								event.preventDefault();
-								updateClassName( style );
-							}
-						} }
-						onMouseEnter={ () => onHoverClassName( styleClassName ) }
-						onMouseLeave={ () => onHoverClassName( null ) }
-						role="button"
-						tabIndex="0"
-						aria-label={ style.label || style.name }
+						className="block-editor-block-styles__item-container"
 					>
-						<div className="editor-block-styles__item-preview block-editor-block-styles__item-preview">
-							<BlockPreviewContent
-								name={ name }
-								attributes={ {
-									...attributes,
-									className: styleClassName,
+						<div
+							className={ classnames(
+								'editor-block-styles__item block-editor-block-styles__item', {
+									'is-active': activeStyle === style,
+								} ) }
+							onClick={ () => updateClassName( style ) }
+							onKeyDown={ ( event ) => {
+								if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+									event.preventDefault();
+									updateClassName( style );
+								}
+							} }
+							onMouseEnter={ () => onHoverClassName( styleClassName ) }
+							onMouseLeave={ () => onHoverClassName( null ) }
+							role="button"
+							tabIndex="0"
+							aria-label={ style.label || style.name }
+						>
+							<div className="editor-block-styles__item-preview block-editor-block-styles__item-preview">
+								<BlockPreviewContent
+									name={ name }
+									attributes={ {
+										...attributes,
+										className: styleClassName,
+									} }
+									innerBlocks={ block.innerBlocks }
+								/>
+							</div>
+							<div className="editor-block-styles__item-label block-editor-block-styles__item-label">
+								{ style.label || style.name }
+							</div>
+						</div>
+						{ onUpdateAutoApplyBlockStyles && (
+							<CheckboxControl
+								className="block-editor-block-styles__item-auto-apply"
+								label={ __( 'Auto apply' ) }
```

Comment:
I feel like this needs better wording. Maybe something more simple like "Use as default".

Also, how well does this work with i18n? This string's translation might easily take up two lines. Does it still look good in that case?

Reply:
> Maybe something more simple like "Use as default".

I thought about that, but the problem is that we call our main style Default, so using "Use as default" would be confusing. We can and should change the current default styles to a name like "Standard" but other blocks may still use "Default" in their names.

> Also, how well does this work with i18n? This string's translation might easily take up two lines. Does it still look good in that case?

Well remembered I will double check with bigger strings.